libafl/src/stages/tuneable.rs

```diff
@@ -231,43 +295,55 @@ where
     }
 }
 
-impl<E, EM, M, Z> TuneableMutationalStage<E, EM, Z::Input, M, Z>
+impl<E, EM, I, M, Z> TuneableMutationalStage<E, EM, I, M, Z>
```

Comment:
What's the reason to have an extra `I` here? Do you use it with different inputs?

Reply:
Without the bound change, I can't call `transforming` due to different bounds.